File: DoitMission_09/Doit09_Java/app/src/main/java/com/example/doit09_java/CustomFragment.java
package com.example.doit09_java;


import android.os.Bundle;

import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import android.widget.EditText;


/**
 * A simple {@link Fragment} subclass.
 */
public class CustomFragment extends Fragment {

    private EditText name;
    private EditText age;
    private Button birthday;
    private Button save;


    public CustomFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View rootView = inflater.inflate(R.layout.fragment_custom, container, false);



        return rootView;
    }

}

File: DoitMission_09/Doit09_Kotlin/app/src/main/java/com/example/doit09_kotlin/CustomFragment.kt
package com.example.doit09_kotlin


import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.EditText
import kotlinx.android.synthetic.main.fragment_custom.*

class CustomFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle? ): View? {
        val rootView =  inflater.inflate(R.layout.fragment_custom, container, false)

        var name = rootView.findViewById(R.id.name) as EditText
        var age = rootView.findViewById(R.id.age) as EditText



        save.setOnClickListener {

        }

        return rootView
    }


}
